lib/letter_opener/tasks/letter_opener.rake

```diff
@@ -0,0 +1,11 @@
+require 'rails/tasks'
+
+namespace :tmp do
+  task :letter_opener do
+    location = LetterOpener.configuration.location
+    location = "tmp/letter_opener" if location.blank?
```

Comment:
I think it's "tmp/letter_opener" by default so we can skip this check, right?

Reply:
I really want to avoid that an empty string translates into `rm -rf /*`.

Reply:
fair enough!

Reply:
After thinking about it a bit more I think it's still pretty dangerous since users can set `/` as location of mails and it'll be considered as `rm -rf /*` then. I suggest to revert it to your original PR and clean things up only for default location configuration. What do you think?

Reply:
Using just `tmp/letter_opener` it's fair enough for me.

I've thought this rake task as an extension of rails tasks (`rake tmp:clear`) [1].
It's only available for rails users (as is loaded by `railtie.rb`) and default path will be used almost every time.

If we want to use `LetterOpener.configuration.location`, I prefer to keep that small guard to protect unsuspecting users


[1] https://github.com/rails/rails/blob/master/railties/lib/rails/tasks/tmp.rake

Reply:
@wacko yeah. Let's just use `tmp/letter_opener`. Using `LetterOpener.configuration.location` is not safe even with that guard because user still can set it to `/`.